TripKitAndroid/src/main/java/com/skedgo/tripkit/TripUpdateApi.kt

```diff
@@ -11,18 +11,23 @@ import kotlinx.coroutines.flow.Flow
 
 interface TripUpdateApi {
 
-  @GET
-  fun fetchUpdateAsync(
-      @Url url: String,
-      @Query("includeStops") includeStops: Boolean = true,
-      @Query("v") v: Int = 12
-  ): Observable<RoutingResponse>
+    @GET
+    fun fetchUpdateAsync(
+        @Url url: String,
+        @Query("includeStops") includeStops: Boolean = true,
+        @Query("v") v: Int = 12
```

Comment:
I won't be requesting any alterations here, but I'm curious to understand the exact meaning of the "v" variable. Is it possible to include some code documentation that clarifies its purpose?

Reply:
Hi @sg-jsonjuliane, this is API requirement. I think this is for the version to use. Also only reformat the code so only spacing was changed here.